CMakeLists.txt

```diff
@@ -505,13 +505,79 @@ MESSAGE(STATUS "")
 
 # -- Dedicated libraw target which does not support multi-threading ---------------------------------------
 
-SET(libraw_LIB_SRCS ${LIBRAW_PATH}/internal/dcraw_common.cpp
-                    ${LIBRAW_PATH}/internal/dcraw_fileio.cpp
-                    ${LIBRAW_PATH}/internal/demosaic_packs.cpp
-                    ${LIBRAW_PATH}/src/libraw_cxx.cpp
-                    ${LIBRAW_PATH}/src/libraw_c_api.cpp
-                    ${LIBRAW_PATH}/src/libraw_datastream.cpp
-   )
+SET(libraw_LIB_SRCS
```

Comment:
This will get out-of-date very quickly. It would be better to use glob patterns together with CONFIGURE_DEPENDS to find files automatically. This should be easy assuming that all files in src/ are part of the library.

Reply:
Done. Thank you for the suggestion, it looks much less verbose now.

Note: the solution is not totally elegant because adds to cmake build phase a step for re-checking globbed directory which can be heavy in common case (see the [note](https://cmake.org/cmake/help/latest/command/file.html#glob)).

Also the `src` directory contains stub implementations which are needed to exclude from the build.

Reply:
Given the small number of source files I don't mind the build-time overhead. It's definitely less likely to break with future releases which is a good thing.
I wasn't aware of the stubs but your solution seems reasonable.